### PR TITLE
add method parent_collection_ids and tests for this method.

### PR DESCRIPTION
**Pull request by blancoj**
add method parent_collection_ids to models/concerns/pcdm_behavior.rb to return all the ids belonging to the parent collections, and add test case to
test this new method.

### CODE REVIEW

File: lib/hydra/pcdm/models/concerns/pcdm_behavior.rb [hunk 1]
```diff
@@ -30,5 +30,9 @@ def parent_collections
       aggregated_by.select(&:pcdm_collection?)
     end
 
+    def parent_collection_ids
+        parent_collections.map(&:id)
```
Comment (mjgiarlo) [inline lib/hydra/pcdm/models/concerns/pcdm_behavior.rb]: By convention, Ruby code is indented two spaces.

File: spec/hydra/pcdm/models/collection_spec.rb [hunk 2]
```diff
@@ -2,16 +2,16 @@
 
 describe Hydra::PCDM::Collection do
 
-  let(:collection1) { Hydra::PCDM::Collection.new }
-  let(:collection2) { Hydra::PCDM::Collection.new }
-  let(:collection3) { Hydra::PCDM::Collection.new }
+  let(:collection1) { Hydra::PCDM::Collection.new}
```
Comment (mjgiarlo) [inline spec/hydra/pcdm/models/collection_spec.rb]: Can you revert the changes to lines 5, 6, 7, 9, and 10?

File: spec/hydra/pcdm/models/collection_spec.rb [hunk 3]
```diff
@@ -26,9 +26,13 @@
         subject.members << object1
         subject.members << object2
         subject.members << collection3
+        subject.save
```
Comment (mjgiarlo) [inline spec/hydra/pcdm/models/collection_spec.rb]: Is the call to `#save` necessary to get IDs?  I see some other specs testing for `child_object_ids` and `child_collection_ids` without needing a save.  Calling `#save` does hurt test performance a bit.

Comment (blancoj) [inline spec/hydra/pcdm/models/collection_spec.rb]: Mike, I cleaned up the code as you suggested in your first 2 comments, as
for the save, the test will fail without it.  Trey mentioned something
about Foreign key relations not being saved, and so that's why it was
needed.  I did not quite understand what he meant.  Should I push my
corrections up?  Thank you!

Comment (mjgiarlo) [inline spec/hydra/pcdm/models/collection_spec.rb]: Go for it, @blancoj!

Comment (terrellt) [inline spec/hydra/pcdm/models/collection_spec.rb]: > Is the call to #save necessary to get IDs?

Yeah, it gets the IDs from the persisted graphs in Fedora - there's no hasParent metadata or anything on the child object.

Comment (terrellt) [inline spec/hydra/pcdm/models/collection_spec.rb]: There should probably be an isolated test for #parent_collection_ids. Something like

```ruby
describe "#parent_collection_ids" do
  it "should return the parent's IDs" do
    subject.members = [object1]
    subject.save!

    expect(object1.parent_collection_ids).to eq [subject.id]
  end
end
```
